tween.js

```diff
@@ -31,7 +31,10 @@ pc.extend(pc, function () {
 
             // add any tweens that were added mid-update
             if (this._add.length) {
-                this._tweens = this._tweens.concat(this._add);
+                for (let i = 0; i < this._add.length; i++) {
+                    if (this._tweens.indexOf(this._add[i]) > -1) continue;
```

Comment:
```suggestion
                    if (this._tweens.indexOf(this._add[i]) > -1) 
                        continue;
                        
```

Reply:
Just wondering about the cost of this if there are a large number of tweens going on 🤔 I can't imagine it would be that bad in real use.

Reply:
I agree, if you're using hundreds tweens at the same time or more, this could be an issue. If someone runs into this, can always modify tweens to use map